docs/configs/janusgraph-cfg.md

```diff
@@ -512,9 +512,9 @@ Meta data to include in storage backend retrievals
 
 | Name | Description | Datatype | Default Value | Mutability |
 | ---- | ---- | ---- | ---- | ---- |
-| storage.meta.[X].timestamps | Whether to include timestamps in retrieved entries for storage backends that automatically annotated entries with timestamps | Boolean | false | GLOBAL |
-| storage.meta.[X].ttl | Whether to include ttl in retrieved entries for storage backends that support storage and retrieval of cell level TTL | Boolean | false | GLOBAL |
-| storage.meta.[X].visibility | Whether to include visibility in retrieved entries for storage backends that support cell level visibility | Boolean | true | GLOBAL |
+| storage.meta.[X].timestamps | Whether to include timestamps in retrieved entries for storage backends that automatically annotated entries with timestamps. If enabled, timestamp can be retrieved by `element.value("~timestamp")`. | Boolean | false | GLOBAL |
```

Comment:
Although I see no reason why e.g. `"~timestamp"` could be renamed in the future, I would still prefer the docs to refer to `ImplicitKey.TIMESTAMP.name()` instead. Otherwise, people would start writing queries which use the hard coded value as proposed here. I think we would be more flexible if that's not the case.

Reply:
Good idea, I'll add it and keep both in the doc. Some might prefer using string literal while others not.

Reply:
The string literal will be updated in the docs, but it will not be updated in the user's queries once they are written.
If a user calls `ImplicitKey.TIMESTAMP.name()` in their code and it doesn't exist anymore, they will get a compiler error. But if they use the string literal, queries could fail silently, which makes it way harder to detedt.

Reply:
Understood. Using ImplicitKey, however, means they need to import this constant. Not sure if this class is auto-imported in the gremlin console but I would imagine in some cases users might encounter this trouble. Now we provide both choices in the doc so that they can choose whichever benefits them.

Anyways, I don't think there is any chance to rename this value, unless we decide to deprecate it in the future (e.g. visibility, which is currently not available in CQL/Hbase/Berkeley/InMemory storage). Renaming could cause backward incompatibility with minimal or no benefit. In case that happens, we will announce it as a breaking change in the changelog so that users would be notified.